tests/testthat/apps/logging/app.R

```diff
@@ -0,0 +1,50 @@
+# Launch the ShinyApp (Do not remove this comment)
+# To deploy, run: rsconnect::deployApp()
+# Or use the blue button on top of this file
+
+if (
+  isTRUE(as.logical(Sys.getenv("TESTTHAT_IS_CHECKING", "false"))) ||
+    isTRUE(as.logical(Sys.getenv("R_COVR", "false")))
+) {
+  library("gsm.app")
+} else {
+  pkgload::load_all(here::here())
+}
+
+group_subset <- c(
+  "0X001",
+  "0X021",
+  "0X103",
+  "0X159",
+  "0X161"
+)
+
+sample_dfResults_subset <- gsm.app::sample_dfResults %>%
+  dplyr::filter(GroupID %in% group_subset)
+sample_dfAnalyticsInput_subset <- gsm.app::sample_dfAnalyticsInput %>%
+  dplyr::filter(GroupID %in% group_subset)
+sample_dfGroups_subset <- gsm.app::sample_dfGroups %>%
+  dplyr::filter(
+    GroupLevel != "Site" | (GroupLevel == "Site" & GroupID %in% group_subset)
+  )
+
+library(shiny.telemetry)
```

Comment:
add `shiny.telemetry` to suggests? or just hold off on that until we use logging in a deployed app?